pgjdbc/src/main/java/org/postgresql/translation/ja.po

```diff
@@ -766,17 +770,17 @@ msgstr ""
 
 #: org/postgresql/jdbc/PgCallableStatement.java:413
 msgid "No function outputs were registered."
-msgstr "関数出力は登録されませんでした。"
+msgstr "関数出力は登録されていません。"
 
 #: org/postgresql/jdbc/PgCallableStatement.java:419
 msgid ""
 "Results cannot be retrieved from a CallableStatement before it is executed."
-msgstr "実行される前に、CallableStatement から結果を得ることはできません。"
+msgstr "実行前の CallableStatement から結果の取得はできません。"
 
 #: org/postgresql/jdbc/PgCallableStatement.java:686
-#, fuzzy, java-format
+#, java-format
 msgid "Unsupported type conversion to {1}."
-msgstr "サポートされないバイナリエンコーディングです: {0}."
+msgstr "{1} への型変換はサポートされていません。"
```

Comment:
It looks like we have off-by-one errors like https://github.com/pgjdbc/pgjdbc/blob/178eecc90643b36c8c5cd423ff311b26733384f2/pgjdbc/src/main/java/org/postgresql/jdbc/PgCallableStatement.java#L686

Arguments are 0-indexed, so it should be `{0}`.

I think we can handle those issues later (and update po files accordingly)